### ISSUE TITLE
[Ban plugins] ANY ban plugins out there is working properly

### ISSUE THREAD

**Issue by ficolas** [keepcalm/BukkitForge#597]
ANY ban plugins out there is working properly. Only default bukkit bans, wich sucks. MaxBans, Ban Management, etc... All the plugins are just not working.

**Comment by Ionaru** [comment]
Essentials?

**Comment by ficolas** [comment]
No, essentials bans is not working properly. It at least doesen't allows users to relog, but temp ban is not working. Your avatar is scaring. @Ionaru

**Comment by alexbegt** [comment]
What build/essentials

**Comment by ficolas** [comment]
@alexbegt Latest avaiable for 1.4.7

**Comment by alexbegt** [comment]
hmm

**Comment by ficolas** [comment]
@alexbegt So far all the ban plugins I used are not working properly. I used the latest version of forgebukkit and the latest version of the plugins (but still for 1.4.7) So far the ONLY working good ban is default bukkit bans.

**Comment by Ionaru** [comment]
Use the default bukkit bans then. It may suck, but it's better then nothing. :)

**Comment by ficolas** [comment]
@Ionaru It's not just they suck, I just don't know how to use them. The ban file doesen't come with any kind of instructions.

**Comment by ficolas** [comment]
Also, I'm using essentials and that overwrites /ban command, so only someone with access to the file can ban becouse of essentials overwrite.

**Comment by Ionaru** [comment]
Essentials ban always worked fine for me, instead of tempban I use /jail and /mute. :P

**Comment by ficolas** [comment]
I prefer ban. Ban is better.

**Comment by ficolas** [comment]
@alexbegt any news?

**Comment by alexbegt** [comment]
@ficolas What is broke now

**Comment by ficolas** [comment]
@alexbegt For me the ban plugin(s) keep doing the same thing. The new build is for 1.4.7 or the next minecraft version? Im getting a ton of strange things with latest builds and might be becouse of that. (Im utill on 1.4.7)
